content/3_Silver/Intro_Sorted_Sets.mdx

```diff
@@ -125,13 +123,39 @@ which can only be used once before calling the `next()` method.
 </Warning>
 
 </JavaSection>
+<PySection>
+
+In Python, we can use `iter()` to obtain the iterator object of any iterable.
+This returns the first element, and using `next()` on the iterator gives you the
+next one. Below, a dictionary is used instead of a set because dictionaries keep
+order.
+
+```py
+nums = {1: None, 7: None, 0: None, 2: None, 4: None}
```

Comment:
i mean why not just have a set if you're gonna make all the keys none

Reply:
Lmao, I wanted to use an "ordered set" so that it shows correct iteration of it's order (1, 7, 0, 2, 4). But if that doesn't really matter that much I can just use a set lol.

Reply:
yeah just use a set